prv_accountant/privacy_random_variables.py

```diff
@@ -4,14 +4,21 @@
 from abc import ABC, abstractmethod
 import numpy as np
 from scipy import integrate
-from numpy import exp, log, sqrt
+from numpy import exp, sqrt
 from numpy import power as pow
 from scipy.special import erfc
 
 M_SQRT2 = sqrt(np.float128(2))
 M_PI = np.pi
 
 
+def log(x):
```

Comment:
As discussed, let's add a test to assert this gives np.log of valid values and still returns np.nan when expected.

Reply:
done